itests/tests/base/test_toasted/envfile.env

```diff
@@ -1 +1 @@
-EXPECTED_COUNT=4
+EXPECTED_COUNT=7
```

Comment:
After debugging, it looks like the `EXEPCTED_COUNT` value has been wrong. 

The existing pollers use a nested loop to read and write to their output files, and the the total record count is only checked outside of the nested loop. Because there's no nested looping for the kafka pooler, the kafka pooler was unable to read more than 4 records from the topic. Setting the `EXPECTED_COUNT` to the desired value allows the kafka poller to read all 7 records from the stream.